### DISCUSSION TITLE
Filter in Dropdown´s

### DISCUSSION THREAD

**Discussion by EduardoR94** [kimai/kimai#3833] [General]
How can you filter so that users only see the projects he works on in dropdowns?
When opening the dropdown, all existing projects are shown and that slows down the processes.

Thank you so much!

**Comment by kevinpapst** [comment]
Teams limit visibility https://www.kimai.org/documentation/teams.html

**Reply by EduardoR94** [reply]
Hi Kevin, thanks for your support.

I have been reading the documentation but I have not yet found a way to only show the user the clients and the projects in which they participate.
(this when creating a new activity).

As I mentioned, all clients and all existing projects are displayed.

How can I get the result I need?

ER

**Reply by kevinpapst** [reply]
By using Teams. Create a team, assign a user, assign the team to the clients/projects/activities.

**Reply by EduardoR94** [reply]
Ok, I think I need to add the team to the client, I'll test it.

Thank you so much!

**Reply by EduardoR94** [reply]
Excellent, I was successful in the tests, thank you very much for the support!

**Reply by kevinpapst** [reply]
It might be a bit confusing when you use it th first time, but great to see that you got it working!
